### ISSUE TITLE
ability to change the first day of the week

### ISSUE THREAD

**Issue by perdalum** [pengx17/logseq-plugin-heatmap#202]
In Europe and probably other places, the first day of the week is Monday. I looked for a configuration to change this, but couldn't find any. This is an enhancement ;-)

**Comment by pengx17** [comment]
@perdalum
Yeah, this could be an enhancement, I think we could add such customizations till Logseq adds plugin options UI support (which could be very soon). I tend to make plugins work out-of-the-box :D

**Comment by pengx17** [comment]
It seems the heatmap library I am using does not support changing start of week. Will refactor with https://github.com/arunghosh/react-grid-heatmap instead.

**Comment by tevino** [comment]
@pengx17 Any plan on this?
